config/zones/BG.yaml

```
@@ -4,16 +4,17 @@ bounding_box:
   - - 28.5580814959
     - 44.2349230007
 capacity:
-  biomass: 80
-  coal: 4365
-  gas: 1371
+  battery storage: 0
+  biomass: 75
```

```suggestion
  biomass: 81
```
Electricity Maps currently group biomass and waste, 75 and 6 MW respectively.